effects/zita1/zitagui.h

```diff
@@ -39,9 +39,9 @@ class ZitaEffectGui : public EffectGui {
       std::vector<QPixmap> pm;
       std::vector<Rotary> rotary;
 
-      int mx, my;
-      int r;
-      qreal oval;
+      int mx{0}, my{0};
+      int r{0};
+      qreal oval{0.};
```

Comment:
`0.`? Isn't `.0` better?

Reply:
I prefer `0.0` for clarity and consistency.

Reply:
changed to 0.0